doc/source/data/batch_inference.rst

```diff
@@ -462,7 +462,7 @@ Models that have been trained with :ref:`Ray Train <train-docs>` can then be use
 
     checkpoint = result.checkpoint
 
-**Step 3:** Use Ray Data for batch inference. To load in the model from the :class:`Checkpoint <ray.air.checkpoint.Checkpoint>` inside the Python class, use one of the :ref:`framework-specific Checkpoint classes <train-framework-catalog>`.
+**Step 3:** Use Ray Data for batch inference. To load in the model from the :class:`Checkpoint <ray.air.checkpoint.Checkpoint>` inside the Python class, use one of the framework-specific Checkpoint classes.
```

Comment:
Is there no link anymore to framework specific Checkpoints? If these are deprecated, what's the recommended way to do prediction with models trained with Ray Train?

Reply:
> If these are deprecated, what's the recommended way to do prediction with models trained with Ray Train?

Also wondering this.

Reply:
Based on @ericl's Slack comment, it sounds like references to framework-specific checkpoints should be removed from the docs. Could we replace "use one of the framework-specific Checkpoint classes." with the current recommendation?

Reply:
Let's decouple the removal from this PR. We'll update the content when we fully quarantined the framework-specific checkpoint classes.

Reply:
> Let's decouple the removal from this PR.

I agree, but this PR makes the removal right? It's no longer documented with this PR merged.